### ISSUE TITLE
 [MAS2.4.6] [Headings and Labels -Welcome] ” Start by testing your bot, My bots ,How to Build a Bot “ are defined in heading  <h3> instead of heading <h2> 

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1822]
**Actual Result:**
” Start by testing your bot, My bots ,How to Build a Bot “are defined in heading  h3 instead of heading h2 present in the welcome screen 

**Expected Result:**
”Start by testing your bot, My bots ,How to Build a Bot “ are should be assigned to h2 heading 
Note: How to build a  bot heading size should be same with start by testing your bot heading  

**Repro Steps:**
Turn on The NVDA 
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened
3. Navigate to the headings using headings mode (H) key 
4. Observe the issue 

**User Impact:**
Users who rely on screen reader will be affected if the headings are defined illogical then the screen then the users will be affected to navigate to the screen.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Application: Bot Framework V4 Emulator
Screen reader: NVDA 

**MAS Reference Link:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0cf80872-800d-4643-90cf-0c1c3d3e6260}
[MAS2.4.6_Welcome.zip](https://github.com/microsoft/BotFramework-Emulator/files/3574231/MAS2.4.6_Welcome.zip)

**Comment by tonyanziano** [comment]
@DesignPolice This fix would require us to change the `HOW TO BUILD A BOT` header to match the other headers on the page since they are at the same organizational level, and must match each other according to the accessibility standards.

Here's how it would look:

**Before:**

<img width="1040" alt="before" src="https://user-images.githubusercontent.com/3452012/64652238-378e9a80-d3d8-11e9-8d6b-8dc358c4829a.png">

**After:**

<img width="1040" alt="after" src="https://user-images.githubusercontent.com/3452012/64652253-3cebe500-d3d8-11e9-9b0a-999584d7149f.png">

Any thoughts on this change?

**Comment by DesignPolice** [comment]
I can live with that, can we make sure that we have a horizontal alignment with the type... either the baseline of the Bot Emulator line or the same line as Version 4
![Screen Shot 2019-09-10 at 2 54 27 PM](https://user-images.githubusercontent.com/45223620/64653288-ec29bb80-d3da-11e9-8beb-f3897b53fbc1.png)

![Screen Shot 2019-09-10 at 2 52 25 PM](https://user-images.githubusercontent.com/45223620/64653224-bedd0d80-d3da-11e9-868f-02f41a299b2a.png)

**Comment by DesignPolice** [comment]
Strong work Showtime.
